accounting/acct.proto

```diff
@@ -0,0 +1,147 @@
+// Accounting for gnXi operations.
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// A protobuffer definition used for logging events reported to gNxI server.
```

Comment:
```suggestion
// A protobuffer definition used for logging events reported to gNMI server.
```

I think? gNxI in this context is a bit too vague, since we don't have a service defined.